.github/workflows/documentation.yml

```diff
@@ -0,0 +1,63 @@
+name: Documentation
+
+on:
+    push:
+        branches:
+            - main
+
+jobs:
+    Generate:
```

Comment:
Isnt standard YAML indentation 2 spaces not 4?

Reply:
Yeah it is, this will break

Reply:
yamlvalidator.com says its fine sooo